packages/salesforcedx-vscode/README.md

```diff
@@ -53,8 +53,10 @@ The Salesforce Extension Pack extension installs these extensions.
    This extension (`salesforcedx-vscode-lightning`) supports Aura component bundles. It uses the HTML language server from VS Code.
 - [Visualforce](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-visualforce)  
    This extension (`salesforcedx-vscode-visualforce`) supports Visualforce pages and components. It uses the Visualforce Language Server and the HTML language server from VS Code.
+- [SOQL](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-soql)  
+   This extension (`salesforcedx-vscode-soql`) enables you to interactively build a SOQL query via a form-based visual editor, view the query as you build, and save the output to a .csv or .json file.
```

Comment:
FYI @sbudhirajadoc

Reply:
FYI, I've copied the text from: [a relative link](packages/salesforcedx-vscode-soql/README.md). I will change it on both places if needed.
One think: it does say anything about the code-editing for SOQL (syntax checks and code-completion)